source/utils/moneyFormatter.ts

```diff
@@ -1,10 +1,18 @@
-const formatDuffs = (
-  duffsAmount: number,
-  decimalPlaces = 8,
+const duffsToDash = (duffsAmount: number): number => {
+  const power = 10 ** 8;
+  const rounded = Math.round(duffsAmount * power) / power;
```

Comment:
Please get rid of rounding, let's make it just `const dashAmount = duffsAmount / 10 ** decimalPlaces;`